From f4bafc14a143359a531d36b56e8a37583b9de148 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Thu, 5 Jul 2018 11:54:07 +0100
Subject: [PATCH 208/249] manifest: Switch to our fork of art

 * Required for adding support for sdm845 platform
   (see Change-Id: Iad0dc93b4e99fe17af16cda5316b06ae4f17aa2c)

Change-Id: Ic9fc7218d74d6d61092ee377525f4af8928c9f1c
---
 default.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/default.xml b/default.xml
index cbc573e..9b69833 100644
--- a/default.xml
+++ b/default.xml
@@ -35,7 +35,7 @@
     <linkfile src="root.bp" dest="Android.bp" />
     <linkfile src="bootstrap.bash" dest="bootstrap.bash" />
   </project>
-  <project path="art" name="platform/art" groups="pdk" remote="aosp" />
+  <project path="art" name="LineageOS/android_art" groups="pdk" />
   <project path="bionic" name="LineageOS/android_bionic" groups="pdk" />
   <project path="bootable/recovery" name="LineageOS/android_bootable_recovery" groups="pdk" />
   <project path="compatibility/cdd" name="platform/compatibility/cdd" groups="pdk" remote="aosp" />
-- 
2.17.1

